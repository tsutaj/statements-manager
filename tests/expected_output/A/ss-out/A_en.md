


## Problem A

### <u>**A+B**</u>

You are given two integers $A$ and $B$. Print the calculation result of $A + B$.

### Input

Input is given from standard input in the following format:

```
$A$ $B$
```

- $1 \leq A, B \leq 10^{9}$

### Output

You have to output the calculation result of $A + B$ to the standard output.


### Sample Input 1
```
3 5
```



### Sample Output 1
```
8
```





### Sample Input 2
```
10000 100
```



### Sample Output 2
```
10100
```








