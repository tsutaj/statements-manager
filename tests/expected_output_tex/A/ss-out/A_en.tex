\documentclass[11pt,a4paper]{article}
\usepackage{amsmath,amssymb}
\usepackage{ascmac}

\begin{document}


\section{Problem A}

\subsection{A+B}

You are given two integers $A$ and $B$. Print the calculation result of $A + B$.

\subsubsection{Input}

Input is given from standard input in the following format:

\begin{screen}
$A$ $B$
\end{screen}

\begin{itemize}
    \item $1 \leq A, B \leq 10^{9}$
\end{itemize}

\subsubsection{Output}

You have to output the calculation result of $A + B$ to the standard output.


\subsection{ Input/Output Samples }



\begin{itembox}[l]{ Sample Input 1 }
3 5

\end{itembox}



\begin{itembox}[l]{ Sample Output 1 }
8

\end{itembox}







\begin{itembox}[l]{ Sample Input 2 }
10000 100

\end{itembox}



\begin{itembox}[l]{ Sample Output 2 }
10100

\end{itembox}







\end{document}