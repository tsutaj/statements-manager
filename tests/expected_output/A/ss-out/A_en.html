<!DOCTYPE html>
<html lang="ja">

<head>
    <title>Problem Statement</title>
    <meta charset="utf-8">
    <link href='http://fonts.googleapis.com/css?family=Noto+Serif' rel='stylesheet' type='text/css'>
    <style type="text/css">
        html {
            color: #222222;
            font-family: "Noto Serif";
        }

        h1 {
            font-size: 16pt;
            border-bottom: solid 2px #e0e0e0;
            margin-bottom: 20px;
        }

        h2 {
            font-size: 14pt;
            border-bottom: solid 2px #e0e0e0;
            margin-bottom: 20px;
            margin-top: 20px;
        }

        h3 {
            font-size: 14pt;
            border-bottom: solid 2px #e0e0e0;
            margin-bottom: 20px;
            margin-top: 20px;
        }

        h4 {
            font-size: 13pt;
            margin-bottom: 20px;
            margin-top: 20px;
        }

        h5 {
            font-size: 12pt;
            margin-bottom: 20px;
            margin-top: 20px;
        }

        b {
            font-weight: bold;
        }

        p {
            font-size: 12pt;
        }

        li {
            font-size: 12pt;
        }

        pre {
            font-size: 11pt;
            padding: 8px;
            border: solid 1px #e0e0e0;
            border-radius: 4px;
        }

        code {
            color: #e83e8c;
        }

        code.language-input-format {
            color: #222222;
        }

        .testcase_pre {
            font-size: 11pt;
            padding: 8px;
            border: solid 1px #e0e0e0;
            border-radius: 4px;
            overflow: auto;
            max-width: 480;
            max-height: 340;
        }

        .error_pre {
            font-size: 11pt;
            padding: 8px;
            border: solid 1px #f0e0e0;
            border-radius: 4px;
            overflow: auto;
            max-height: 340;
            background-color: #ffeeee;
        }

        .table tbody>tr>td.vert-aligned {
            vertical-align: middle;
        }
    </style>
    <script type="text/x-mathjax-config">
        MathJax.Hub.Config({
            tex2jax: {
                inlineMath: [["$", "$"]],
                processEscapes: true,
                processClass: 'input-format|language-input-format'
            }
        });
    </script>
    <script type="text/javascript"
        src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js?config=TeX-AMS_CHTML">
        </script>
</head>

<body>
    
    <div >
        <h2>Problem A</h2>
<h3><u><strong>A+B</strong></u></h3>
<p>You are given two integers $A$ and $B$. Print the calculation result of $A + B$.</p>
<h3>Input</h3>
<p>Input is given from standard input in the following format:</p>
<pre class="input-format"><code class="language-input-format">$A$ $B$
</code></pre>
<ul>
<li>$1 \leq A, B \leq 10^{9}$</li>
</ul>
<h3>Output</h3>
<p>You have to output the calculation result of $A + B$ to the standard output.</p>
<h3>Sample Input 1</h3>
<pre class="input-format"><code class="language-input-format">3 5
</code></pre>
<h3>Sample Output 1</h3>
<pre class="input-format"><code class="language-input-format">8
</code></pre>
<h3>Sample Input 2</h3>
<pre class="input-format"><code class="language-input-format">10000 100
</code></pre>
<h3>Sample Output 2</h3>
<pre class="input-format"><code class="language-input-format">10100
</code></pre>
    </div>
    
</body>

</html>